operator/cmd/mesh/manifest-apply.go

```diff
@@ -152,36 +199,41 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, maArgs *manifestApplyAr
 //  dryRun  all operations are done but nothing is written
 //  verbose full manifests are output
 //  wait    block until Services and Deployments are ready, or timeout after waitTimeout
-func ApplyManifests(setOverlay []string, inFilenames []string, force bool, dryRun bool, verbose bool,
-	kubeConfigPath string, context string, wait bool, waitTimeout time.Duration, l *Logger) error {
+func ApplyManifests(args *runApplyManifestArgs) error {
 
-	ysf, err := yamlFromSetFlags(setOverlay, force, l)
+	ysf, err := yamlFromSetFlags(args.set, args.force, args.l)
 	if err != nil {
 		return err
 	}
 
-	kubeconfig, err := manifest.InitK8SRestClient(kubeConfigPath, context)
+	kubeconfig, err := manifest.InitK8SRestClient(args.kubeConfigPath, args.context)
 	if err != nil {
 		return err
 	}
-	manifests, iops, err := GenManifests(inFilenames, ysf, force, kubeconfig, l)
+	manifests, iops, err := GenManifests(args.inFilenames, ysf, args.force, kubeconfig, args.l)
 	if err != nil {
 		return fmt.Errorf("failed to generate manifest: %v", err)
 	}
+	global := iops.Values["global"]
+	clusterID := ""
+	v, ok := global.(map[string]interface{})
+	if ok {
+		clusterID = fmt.Sprint(v["clusterID"])
+	}
```

Comment:
Why does `clusterID` need to be extracted here? It's already in `iops` which is passed to `manifest.ApplyAll`.

Reply:
moved into  manifest.ApplyAll